packages/slice-machine/server/src/api/services/getEnv.ts

```diff
@@ -88,12 +89,12 @@ export default async function getEnv(
   // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
   const mockConfig = getMockConfig(cwd);
 
-  const client = initClient(
-    cwd,
-    prismicData.value.base,
-    repo,
-    prismicData.value.auth
-  );
+  const base = preferWroomBase(manifestInfo.content.apiEndpoint);
+  const auth = prismicData.value.base.startsWith(base)
```

Comment:
Good, but I would have done that in the function that retrieve the `prismicData`.
By passing the manifest to it.
That way we ensure that whenever we need the `.prismic` auth informations, we get the manifest first and it is used to know if we're logged in or not.